airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -694,29 +710,23 @@ def _build_dataflow_job_name(job_name: str, append_job_name: bool = True) -> str
         return safe_job_name
 
     @staticmethod
-    def _build_cmd(variables: Dict, label_formatter: Callable, project_id: str) -> List[str]:
-        command = [
-            "--runner=DataflowRunner",
-            "--project={}".format(project_id),
-        ]
-        if variables is None:
-            return command
-
+    def _options_to_args(variables: Dict):
+        if not variables:
+            return []
         # The logic of this method should be compatible with Apache Beam:
         # https://github.com/apache/beam/blob/b56740f0e8cd80c2873412847d0b336837429fb9/sdks/python/
         # apache_beam/options/pipeline_options.py#L230-L251
+        args: List[str] = []
         for attr, value in variables.items():
-            if attr == 'labels':
-                command += label_formatter(value)
-            elif value is None:
-                command.append(f"--{attr}")
+            if value is None:
```

Comment:
nit: this if and the next elif take the same action and could be combined
```suggestion
            if value is None or (isinstance(value, bool) and value):
```

Reply:
Done :)